src/device/Heater.cpp

```diff
@@ -129,3 +202,25 @@ float Heater::getTemperature() {
 HeaterStatus Heater::getHeaterState() const {
   return heaterStatus;
 }
+
+String Heater::deviceAddressToString(DeviceAddress deviceAddress) const {
+  String str = "";
+  for (uint8_t i = 0; i < 8; i++){
+    if( deviceAddress[i] < 16 ) str += String(0, HEX);
+    str += String(deviceAddress[i], HEX);
+  }
+  return str;
+}
+
+float Heater::getHumidity() {
+  return weatherSensor.getHumidity();
+}
+
+float Heater::dewPoint(float humidity, float temperature) const {
+  return 243.04 * (log(humidity / 100.0) + ((17.625 * temperature) / (243.04 + temperature))) 
```

Comment:
If this is a well-known equation than there should be a comment with more informations (link to wikipedia?)

Reply:
Good point